### PR TITLE
Feature/fgpb 101

### PR DESCRIPTION
**Pull request by MatthewWaanders**

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -5561,6 +5561,50 @@
...
+            "nl": "Druk 6 keer op de Druk Knop"
+          }
+        },
+        "associationGroups": [
+          1
+        ],
+        "associationGroupsOptions": {
+          "1": {
+            "hint": {
+              "en": " Lifeline  reports the device status and allows for assigning single device only (main controller by default).\r\n"
```
Comment (RobinBol) [inline app.json]: Double space (probably also with all other drivers?)

File: drivers/FGPB-101/device.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+class Button extends ZwaveDevice {
+	
+	onMeshInit() {
+		this.registerCapability('measure_battery', 'BATTERY');
+		this._onButtonTrigger = new Homey.FlowCardTriggerDevice("FGPB-101").register().registerRunListener((args, state, callback) => {
+			if(state && args &&
+				state.hasOwnProperty('scene') &&
+				args.hasOwnProperty('scene')) {
+					return callback(null, state.scene === args.scene);
+            }
```
Comment (RobinBol) [inline drivers/FGPB-101/device.js]: Strange indentation, maybe add eslint --fix as pre-commit action

File: drivers/FGPB-101/driver.settings.compose.json [hunk 3]
```diff
@@ -0,0 +1 @@
+[]
```
Comment (RobinBol) [inline drivers/FGPB-101/driver.settings.compose.json]: No settings? Maybe remove the file then.

File: app.json (hunk 1 above)
Comment (MatthewWaanders) [reply app.json]: Fixed in driver.compose.json, will rebuild app on sdk2 branch (needs to happen anyway)

**Pull request merged**